substrate/frame/src/lib.rs

```diff
@@ -227,7 +227,7 @@ pub mod runtime {
 			<<Signature as traits::Verify>::Signer as traits::IdentifyAccount>::AccountId;
 
 		/// The block-number type, which should be fed into [`frame_system::Config`].
-		pub type BlockNumber = u32;
+		pub type BlockNumber = u64;
```

Comment:
This is not a testing type.

Reply:
Got it, I thought the "opinionated types aliases commonly used in runtimes" for `BlockNumber` would need to change as well since we're unifying the test type for `BlockNumber`. Reverted this change.

Reply:
Issue is that it is exposed for features `runtime` and `std`. If it was just `std` then it would indeed be testing-only.  
But like this it can also be used in normal runtimes. Thanks.